_posts/2023-02-19-a-lean-dataops-platform.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: post
+inject_optimize_tag: true
+title: "A Lean Modern Data Platform"
+description: ""
+date: 2023-02-22
+highlight: monokai
+author: Matthew Carter
+tags: [snowflake, data-ops, data platform, fivetran, dbt, dagster]
+---
+
+# Building a Lean ‘Modern’ Data Platform
+## Introduction
+Right now, everyone wants to be data driven and there has been an explosion of tooling to help people do that. However, to become data driven with things like dashboards, reporting, AI and ML, it is paramount that these soon-to-be groan inducing buzzwords are built upon a solid foundation of quality, aggregated and easy to access data. A well engineered data platform is, in my opinion, the answer to this problem. For companies looking to effectively aggregate and analyze large amounts of data, building a platform is essential. *Excel sheets can only get you so far.* With a data platform, businesses generally can gain insights into their operations, customer behavior, and market trends. This knowledge can help them make informed decisions and ultimately move closer to becoming data driven. 
+
+For Mechanical Rock, we wanted to build an internal data platform with a few key objectives using data-ops coding practices. 
```

Comment:
I guess using data-ops coding practices it's the way of delivering however key objective would probably be something like using patterns that allows for good security, maintainability and scalability of the solution.

Reply:
Hey I don't really understand what you mean by this. 
The platform as I understand it has a could things it needs to be able to do (a few key objectives)
The way the platform was/is developed is with data-ops practices (CICD, Infrastructure as code)

Reply:
Oh your sentence reads as the objectives were dataOps coding practices. `with a few key objectives using data-ops coding practices`. If that's not the intent maybe that sentence needs a bit of rewording

Reply:
yes agreed

Reply:
completed 🫡